src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -232,6 +250,15 @@ const Notebooks = _.flow(
     this.setState({ notebooks: _.reverse(_.sortBy('updated', notebooks)) })
   })
 
+  refreshApps = _.flow(
```

Comment:
the above refresh should be called refreshNotebooks to match this name convention